File: 10 - Python/Exercícios/02 - Estrutura Condicional/Exercício 04.py
#Exercício - Operadora

minutos = int(input("Digite a quantidade de minutos: "))

planoBasico = 50

if minutos <= 100:
    print("Valor a pagar: R$", planoBasico)
else:
    excedente = planoBasico + 2 * (minutos - 100)
    print("Valor a pagar:  R$", excedente)

File: 06 - C/Exercícios/02 - Estrutura Condicional/Exercício 04.c
//Exercício - Operadora

#include <stdio.h>

int main() {

    int minutos;
    double planoBasico, excedente;

    printf("Digite a quantidade de minutos: ");
    scanf("%d", &minutos);

    planoBasico = 50;

    if (minutos <= 100){
        printf("Valor a pagar: R$%.2lf", planoBasico);
    }
    else{
        excedente = planoBasico + 2 * (minutos - 100);
        printf("Valor a pagar: R$%.2lf", excedente);
    }

    return 0;

}